core/client/app/templates/post-settings-menu.hbs

```diff
@@ -13,7 +13,12 @@
                 <label for="url">Post URL</label>
                 {{#if model.isDraft}}
                 <a class="post-preview-link" target="_blank" href="{{model.previewUrl}}">
-                    Preview<span class="icon-external"></span>
+                    Preview <i class="icon-external"></i>
+                </a>
+                {{/if}}
+                {{#if model.isPublished}}
```

Comment:
I guess "isPublished" could be the default and "else" all other states get the preview - that seems relatively sane